include/aspect/particle/interface.h

```diff
@@ -0,0 +1,64 @@
+/*
+ Copyright (C) 2015 - 2022 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file LICENSE.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _aspect_particle_interface_h
+#define _aspect_particle_interface_h
+
+#include "aspect/plugins.h"
+
+namespace aspect
+{
+  namespace Particle
+  {
+    /**
+     * An abstract base class defining methods and member variables
```

Comment:
This class is not abstract (https://en.cppreference.com/w/cpp/language/abstract_class)
```suggestion
     * A base class defining methods and member variables
```